packages/ods-react/src/components/skeleton/src/components/skeleton/Skeleton.tsx

```diff
@@ -14,7 +14,8 @@ const Skeleton: FC<SkeletonProp> = forwardRef(({
       className,
     )}
     ref={ref}
-    {...props}>
+    {...props}
+    aria-hidden="true">
```

Comment:
even if `aria-hidden` should always be true, it may be better to set it above the props extend (people may want to be creative for some reason with their usage of skeleton)